src/lib/components/OpenWebSearchResults.svelte

```diff
@@ -88,6 +123,16 @@
 </details>
 
 <style>
+	details[open] .content {
+		animation-name: grow;
+		animation-duration: 300ms;
+		animation-delay: 0ms;
```

Comment:
the animation only works for the first time, And there is no animation when you close it or open it for the 2+ time

Reply:
I removed it completly.